packages/entities/entities-shared/src/components/entity-base-table/EntityBaseTable.vue

```diff
@@ -350,6 +353,9 @@ const handleSortChanged = (sortParams: TableSortParams): void => {
 }
 
 const { setTablePreferences, getTablePreferences } = useTablePreferences()
+
+const tablePreferences = ref<UserTablePreferences>(getTablePreferences(props.preferencesStorageKey))
```

Comment:
The `props.preferencesStorageKey` values currently being passed are DEFINITELY not unique enough for this to be used. I thought we had talked about using the `cacheIdentifier` instead along with a prefix like `tablePreferences-`.

Reply:
Re-thinking

Reply:
Now using unique key for localStorage of user's table preferences across tables, orgs and users